contrib/pipelines/secadvisor/README.md

```diff
@@ -1,34 +1,85 @@
 # Security Advisor Pipeline
 
-Implements the IBM Security Advisor pipeline
+The IBM Security Advisor pipeline extracts flow data from skydive, filters out
+external traffic, transforms flow records by adding status information, and
+flow classification (egress, ingress, internal), batches flows records together
+into streams (by time or count), then encodes these blocks in JSON form, and
+compresses using GZIP and finally stores as objects into any S3 compatible
+target.
 
+## Deploy Skydive
 
-## Setup Skydive
-
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
 
 ```
 make
-cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+sudo $(which skydive) allineone -c etc/skydive.yml.default
 ```
 
 ## Setup Minio
 
-Install and run minio (an AWS S3-like object store).
+For running tests on a local machine, it is possible to set up a local Minio
+object store.
+
+First run the minio server:
 
-## Setup Pipeline
+```
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
```

Comment:
complicated .... why not just `docker run -d -p 9000:9000 --name minio -e "MINIO_ACCESS_KEY=user" -e "MINIO_SECRET_KEY=password" -v /mnt/data:/data -v /mnt/config:/root/.minio minio/minio server /data`

Or even more simple ... just reference to https://docs.min.io/docs/minio-docker-quickstart-guide

Reply:
no. that leaves it too open ended while i want a concreate sequance